src/common/system.rs

```diff
@@ -847,6 +819,35 @@ impl System {
         SystemInner::distribution_id_like()
     }
 
+    /// Provides kernel version following this string format:
+    ///
+    /// "<code>[distribution_id] OS Build [kernel_version]</code>"
```

Comment:
This explanation is no longer accurate.

It would be helpful to include a table of example platforms like https://docs.rs/sysinfo/0.33.1/sysinfo/struct.System.html#method.kernel_version